.github/docs/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+set -u
+
+SSH_PATH="/root/.ssh"
+
+mkdir "$SSH_PATH"
+touch "$SSH_PATH/known_hosts"
+
+echo "$DEPLOY_SSH_KEY" > "$SSH_PATH/id_rsa"
+
+chmod 700 "$SSH_PATH"
+chmod 600 "$SSH_PATH/known_hosts"
+chmod 600 "$SSH_PATH/id_rsa"
+
+eval $(ssh-agent)
+ssh-add "$SSH_PATH/id_rsa"
+
+ssh-keyscan -t rsa "github.com" >> "$SSH_PATH/known_hosts"
+echo "StrictHostKeyChecking no" >> "$SSH_PATH/config"
+
+# Change into the build directory
+cd $BUILD_DIR
+
+git config --global user.name "jonwinton" && \
+git config --global user.email "jonwinton@users.noreply.github.com" && \
```

Comment:
Since we can't use the [environment variables](https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/) in this case `GITHUB_ACTOR`.  Can we store this value on a secret?
What you think? @jonwinton

Reply:
Good catch! Fixing!